mapviz_plugins/src/marker_plugin.cpp

```diff
@@ -328,44 +407,90 @@ namespace mapviz_plugins
         {
           if (marker.transformed)
           {
-            glColor4f(marker.color.redF(), marker.color.greenF(), marker.color.blueF(), 1.0f);
+            glColor4d(marker.color.redF(), marker.color.greenF(), marker.color.blueF(), marker.color.alphaF());
```

Comment:
Not exactly related to this change, but I the alpha channel for markers wasn't being respected here.